src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -609,6 +614,97 @@ public Response updateDraftVersion( String jsonBody, @PathParam("id") String id,
             
         }
     }
+  
+    @GET
+    @Path("{id}/versions/{versionId}/metadata")
```

Comment:
Was there supposed to be an extra entry in the path for this API call - something like like `"{id}/versions/{versionId}/metadata/jsonld"`?

Reply:
(or is the idea to have the same `.../metadata`, and to have the client request jsonld by supplying an `Accept:` header for it? 
(that should be legal too, right?...)

Reply:
I was going for the latter - line 620 means this method picks up the Accept: application/json-ld and the other gets anything else.

Reply:
This may be a cleaner way  of doing it too, not having to declare the format an extra parameter. I guess the only disadvantage is that you can't bookmark a standalone URL for a specific content type (?). But I'm assuming it's ok for this specific format.

Reply:
Yeah - I think it is ~common. FWIW: I do this in the newmetrics PR (#7178) as well to let you select csv or json and I set the csv as the default so if you bookmark/request via the browser, you get that. The default here would still be the json (not json-ld).